### PR TITLE
adding scripts

### PR DESCRIPTION
**Pull request by NedraMekni**
## Description
Preparing scripts for data collection (PDB of protein-ligand complex, IC50 values)

## Todos
Scripts review
/scripts/get_structures.py

## Status
- [ ] Ready to go

### CODE REVIEW

File: InterGraph/interaction_graph.py [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+from rdkit import Chem
+from scipy.spatial.distance import cdist
+from itertools import product
+from openbabel import openbabel
+
+pdb_raw_d = "../data/" + "pdb_raw"
+
+
+def file_path():
+    global pdb_raw_d
```
Comment (wiederm) [inline InterGraph/interaction_graph.py:21]: I am not super sure why you call global here --- can you give some context on this decision?

File: InterGraph/interaction_graph.py [hunk 2]
```diff
@@ -0,0 +1,404 @@
...
+    if not os.path.exists("../data/" + "data"):
+        os.makedirs("../data/" + "data")
+    if not os.path.exists("../data/" + "pdb_raw"):
+        os.makedirs("../data/" + "pdb_raw")
+
+
+def get_pdb_components(pdb_id):
```
Comment (wiederm) [inline InterGraph/interaction_graph.py:28]: can you add type hints here, in this case it would be:
` def get_pdb_components(pdb_id:str)->None:`

Comment (wiederm) [reply InterGraph/interaction_graph.py:28]: and, maybe a short line that explains what this function does

File: InterGraph/interaction_graph.py [hunk 3]
```diff
@@ -0,0 +1,404 @@
...
+    global pdb_raw_d
+
+    pdb_r = urllib.request.urlretrieve(
+        "https://files.rcsb.org/download/{}.pdb".format(pdb_id),
+        "{}/{}.pdb".format(pdb_raw_d, pdb_id),
+    )
+
+
+def parsePDB(lig, filename):
```
Comment (wiederm) [inline InterGraph/interaction_graph.py:37]: `def parsePDB(lig:str, filename:str)->Tuple[list, list]:`

File: InterGraph/interaction_graph.py [hunk 4]
```diff
@@ -0,0 +1,404 @@
...
+        mol_graphs_crystal_6A = {}
+        contacts_6A = GetAtomContacts(
+            protein_path, c_mol, Atom_Keys, distance_cutoff=6.0
+        )
+        graph_c_6 = mol_to_graph(c_mol, contacts_6A, Atom_Keys)
+        mol_graphs_crystal_6A[pdb] = graph_c_6
+        return mol_graphs_crystal_6A
+
+
+def GetAtomType(atom):
```
Comment (wiederm) [inline InterGraph/interaction_graph.py:101]: I'd recommend to use the PEP 8 style guide recommendation for naming functions:
https://peps.python.org/pep-0008/#function-and-variable-names

Comment (NedraMekni) [reply InterGraph/interaction_graph.py]: I've committed and pushed the suggested changes in this pull request branch

**Comment by wiederm** [comment]
but I just realized that the tests are failing on the CI: https://github.com/NedraMekni/InterGraph/runs/6625928348?check_suite_focus=true

It seems that you haven't specified all the required packages:
https://github.com/NedraMekni/InterGraph/blob/main/devtools/conda-envs/test.yaml

And, looking at `test.yaml`: I would only add the packages that are truly needed, and without version if possible. E.g.:
https://github.com/wiederm/endstate_rew/blob/main/devtools/conda-envs/test_env.yaml

**Pull request merged**